install/build_on_vm.sh

```diff
@@ -130,11 +110,10 @@ gcloud compute ssh "$machine_name" -- bash << EOF
   fi
   git clone -b "$branch" https://github.com/pagespeed/mod_pagespeed.git
   cd mod_pagespeed
-  install/build_release.sh $@
+  install/build_release.sh ${remaining_arguments[@]}
```

Comment:
Oh, was the trailing quotes on remaining_arguments to deal with set -u here?

Maks' solution for this was to test the size of the array first, a la:
if [ "${#remaining_arguments[@]}" -gt 0 ]; then
  install/build_release.sh ${remaining_arguments[@]}
else
  install/build_release.sh
fi

The fancier approach I used in centos/install_required_packages.sh is:
install/build_release.sh ${remaining_arguments[@]+"${remaining_arguments[@]}"}

(from http://stackoverflow.com/questions/7577052/bash-empty-array-expansion-with-set-u)